source/features/sticky-comment-header.css

```diff
@@ -0,0 +1,55 @@
+html:not([rgh-OFF-sticky-comment-header]) {
+	/* Issue body header */
+	[class^="IssueBodyHeader-module__IssueBodyHeaderContainer"],
+	/* Issue comment header */
+		[data-testid='comment-header'],
+	/* PR body/comment header */
+		.timeline-comment-header {
+		position: sticky;
+		top: calc(var(--base-sticky-header-height, 0px) + 56px);
+		z-index: 1;
+	}
+
+	/* Added by `show-names` */
+	.rgh-viewer-did-author,
+	.timeline-comment.current-user .timeline-comment-header,
+	[class*="IssueBodyHeader-module__viewerDidAuthor"] {
+		background: linear-gradient(var(--bgColor-accent-muted, fuchsia)), var(--rgh-background) !important;
+	}
+
+	/* Show menus on top of headers */
+	.js-timeline-item:has(details:open) .timeline-comment-header {
+		z-index: 2;
+	}
+
+	[class^="IssueBodyHeader-module__IssueBodyHeaderContainer"],
+	[data-testid="comment-header"] {
+		border-radius: 0;
+	}
+
+	/* Issue body container */
+	[data-testid="issue-body"] [class^="Box"]:has(> #issue-body-viewer),
```

Comment:
@kovsu it feels like we just dropped a selector just like this (box:has) because it was causing perf issues.